src/Components/card/card.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Spoiler from "../spoiler/spoiler.js";
+import "./card.css";
+import data from "./data.js";
+import girlPicture from "./illustration-woman-online-desktop.svg";
+import bgPicture from "./bg-pattern-desktop.svg";
+import cubePicture from "./illustration-box-desktop.svg";
+
+const Card = () => {
+    const elements = data.map( (item) => {
+        if ( typeof item === 'object' && isEmpty(item) ){ 
+            return (
+                <Spoiler {...item}/>
```

Comment:
Each element returned from `map` should have unique `key` prop https://reactjs.org/docs/lists-and-keys.html#keys